### DISCUSSION TITLE
Error running kube-hunter scan using kubectl plugin

### DISCUSSION THREAD

**Discussion by PatrickSplice** [aquasecurity/starboard#386] [Questions & Help] [answered]
Trying to run the kube-hunter scan and receiving this error, which I cannot find any reference to in searches.

`kubectl starboard scan kubehunterreports`
`error: property kube-hunter.quick not set`

I was able to run this before. Behaving strangely now.

**Comment by danielpacak** [comment, marked as answer]
This is new param added in v0.9.0 to limit the range of ports to be scanned by kube-hunter. If you upgrade from the previous version you'd need to patch the starboard ConfigMap and add this param with the default value of false manually. See https://aquasecurity.github.io/starboard/v0.9.0/settings/ for reference.

**Reply by PatrickSplice** [reply]
That worked. Thanks. Forgot I had upgraded. Should have checked the settings doc.

**Reply by mlevesquedion** [reply]
Hey @danielpacak, chiming in here since I added this config property in #331.

This looks like it can be confusing to users who are upgrading. A significant portion of users may not care about this particular setting, so maybe it should *not* be required?

**Reply by danielpacak** [reply]
I agree that we could have made this param optional, but on the other hand it's not that big deal to update config manually at this early stage of Starboard development. Anyway I'd be happy to accept a PR if you want to make it optional.
